src/nc/core/likec/Simplifier.cpp

```diff
@@ -400,7 +416,45 @@ std::unique_ptr<MemberAccessOperator> Simplifier::simplify(std::unique_ptr<Membe
 
 std::unique_ptr<Expression> Simplifier::simplify(std::unique_ptr<Typecast> node) {
     node->operand() = simplify(std::move(node->operand()));
-
+    /*
+     * reinterpret_cast<int16_t>(*reinterpret_cast<uint16_t*>(&edi2))
+     * will become *reinterpret_cast<int16_t*>(&edi2)
+     */
+    if(node->operand()->is<UnaryOperator>()
+                                          && node->type()->isInteger() &&
```

Comment:
Indentation here and in many other places is completely random and inconsistent. Could you fix it?